packages/cli/src/util/config/get-default.ts

```diff
@@ -1,75 +1,15 @@
 import { AuthConfig, GlobalConfig } from '../../types';
 
-export const getDefaultConfig = async (existingCopy?: GlobalConfig | null) => {
-  let migrated = false;
-
-  const config: GlobalConfig = {
-    _: 'This is your Vercel config file. For more information see the global configuration documentation: https://vercel.com/docs/configuration#global',
-    collectMetrics: true,
-  };
-
-  if (existingCopy) {
-    const keep = [
-      '_',
-      'currentTeam',
-      'desktop',
-      'updateChannel',
-      'collectMetrics',
-      'api',
-      // This is deleted later in the code
-    ];
-
-    try {
-      const existing = Object.assign({}, existingCopy);
-      // @ts-ignore
-      const sh = Object.assign({}, existing.sh || {});
-
-      Object.assign(config, existing, sh);
-
-      for (const key of Object.keys(config)) {
-        if (!keep.includes(key)) {
-          // @ts-ignore
-          delete config[key];
-        }
-      }
-
-      if (typeof config.currentTeam === 'object') {
-        // @ts-ignore
-        config.currentTeam = config.currentTeam.id;
-      }
-
-      // @ts-ignore
-      if (typeof config.user === 'object') {
-        // @ts-ignore
-        config.user = config.user.uid || config.user.id;
-      }
-
-      migrated = true;
-    } catch (err) {}
-  }
-
-  return { config, migrated };
+export const defaultGlobalConfig: GlobalConfig = {
+  '// Note':
+    'This is your Vercel config file. For more information see the global configuration documentation: https://vercel.com/docs/configuration#global',
```

Comment:
This link is redirecting, should it be updated too?

Reply:
Whoops, removed.